docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -102,6 +103,22 @@ specify a browser alias that consists of the `{browser-provider-name}` prefix an
 testcafe "saucelabs:Chrome@52.0:Windows 8.1" tests/sample-fixture.js
 ```
 
+### Starting browser with arguments
+
+If you need to pass arguments for specified browser, you should write them after browser alias and surround this construction with quotation marks:
```

Comment:
If you need to pass arguments for the specified browser, you should write them right after browser alias. Surround the browser call and its arguments with quotation marks: